cluster/juju/layers/kubernetes-worker/build-cni-resources.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -eux
+
+CNI_VERSION="${CNI_VERSION:-v0.7.1}"
+ARCH="${ARCH:-amd64 arm64 s390x}"
+
+build_script_commit="$(git show --oneline -q)"
+temp_dir="$(readlink -f build-cni-resources.tmp)"
+rm -rf "$temp_dir"
+mkdir "$temp_dir"
+(cd "$temp_dir"
```

Comment:
I like these subshells over pushd/popd, which is what I would have done initially.